src/prashant/CalculatorWithReturn.java

```diff
@@ -0,0 +1,51 @@
+//Write a class called calculator with following functionality.
+//a. add
+//b. subtraction
+//c. division
+//d. multiplication
+//e. printTotalAnswer.
+//
+//Method a,b,c,d should have 2 parameters and return int value.
+//printTotalAnswer method should have 4 parameters and doesnt return anything.
+//Note : do not create any instance variable.
+
+package prashant;
+
+public class CalculatorWithReturn {
+
+	int add(int x, int y) {
+		int addition = x + y;
+		return addition;
+	}
+
+	int sub(int x, int y) {
+		int substraction = x - y;
+		return substraction;
+	}
+
+	int division(int x, int y) {
+		int div = x / y;
```

Comment:
you can handle one condition, when y = 0

Reply:
@harshvegada agreed